src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -106,16 +130,41 @@ private void validateCeasedDate(OverseasEntitySubmissionDto overseasEntitySubmis
             if (UtilsValidators.isNotNull(trustCeasedDate, qualifiedFieldName, errors, loggingContext)) {
                 DateValidators.isDateInPast(trustCeasedDate, qualifiedFieldName, errors, loggingContext);
                 DateValidators.isCeasedDateOnOrAfterCreationDate(trustCeasedDate, creationDate, qualifiedFieldName, errors, loggingContext);
-            }
+
+                if (trustDataDto.getIndividuals() != null && !trustDataDto.getIndividuals().isEmpty()) {
+                    DateValidators.isCeasedDateOnOrAfterIndividualsDateOfBirth(trustCeasedDate, trustDataDto.getIndividuals(), qualifiedFieldName, errors, loggingContext);
+                }
+           }
         }
     }
 
-    private boolean isTrustStillRequired(OverseasEntitySubmissionDto overseasEntitySubmissionDto, String trustId) {
+    private boolean noBeneficialOwnersArePresent(OverseasEntitySubmissionDto overseasEntitySubmissionDto) {
+        boolean allIndividualsDisassociated = overseasEntitySubmissionDto.getBeneficialOwnersIndividual() != null
+                && overseasEntitySubmissionDto.getBeneficialOwnersIndividual().stream().anyMatch(
```

Comment:
it looks like we should be checking if all individuals are disassociated but using anyMatch() will return true if 1 matches, not if all match.

Reply:
Good spot. Changed to check all matches.